Ll firepower rest api105 json

ReneeBornstein opened the pull request:
created titles for table of contents

annegentle commented on labs/firepower-restapi-104/firepower-restapi-104.json:
```diff
@@ -30,9 +30,11 @@
     }],
     "files": [{
-        "title": "1.md"
+       "title": "FMC - REST - API - 104",
```
Don't need these dashes. Also I believe there could be spelling errors in this file, can you check?

annegentle commented on labs/firepower-restapi-105/firepower-restapi-105.json:
```diff
@@ -30,9 +30,11 @@
     "files": [{
-        "title": "1.md"
+        "file": "1.md",
+        "title": "FMC - REST - API - 105"
```
Same here, don't need dashes. FMC REST API 105 is sufficient.

annegentle reviewed (changes requested):
I've updated the first three labs JSON files I believe in the master branch, so you can change 104 and 105.

ReneeBornstein commented:
Okay, I wasn’t sure, and was being consistent. I will change them, and do all of the others properly.

BTW, the reason I’ve only done a few today is that I was also trying to redo the document explaining the tasks for editing each kind of doc. For the JSON doc, it takes 17 steps to get done properly. I’ve attached that section to show you what I’ve written, and make sure you’re okay with it.

From: Anne Gentle <notifications@github.com>
Reply-To: CiscoDevNet/fmc-rest-api <reply@reply.github.com>
Date: Monday, July 2, 2018 at 9:28 AM
To: CiscoDevNet/fmc-rest-api <fmc-rest-api@noreply.github.com>
Cc: "Renee Bornstein -X (rbornste - INSIGHT GLOBAL INC at Cisco)" <rbornste@cisco.com>, Author <author@noreply.github.com>
Subject: Re: [CiscoDevNet/fmc-rest-api] Ll firepower rest api105 json (#53)


@annegentle requested changes on this pull request.

I've updated the first three labs JSON files I believe in the master branch, so you can change 104 and 105.

________________________________

In labs/firepower-restapi-104/firepower-restapi-104.json<https://github.com/CiscoDevNet/fmc-rest-api/pull/53#discussion_r199552787>:

> @@ -30,9 +30,11 @@

         "title": "Cisco Security REST API"

     }],

     "files": [{

-        "title": "1.md"

+       "title": "FMC - REST - API - 104",

Don't need these dashes. Also I believe there could be spelling errors in this file, can you check?

________________________________

In labs/firepower-restapi-105/firepower-restapi-105.json<https://github.com/CiscoDevNet/fmc-rest-api/pull/53#discussion_r199552863>:

> @@ -30,9 +30,11 @@

         "title": "Cisco Security REST API"

     }],

     "files": [{

-        "title": "1.md"

+        "file": "1.md",

+        "title": "FMC - REST - API - 105"

Same here, don't need dashes. FMC REST API 105 is sufficient.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/CiscoDevNet/fmc-rest-api/pull/53#pullrequestreview-133694542>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AlQdkCjReqPowkRdHQsj_EFLafgoNjvQks5uCko8gaJpZM4U_dLw>.

ReneeBornstein committed:
Update firepower-restapi-102.json

removed the dashes in titles

ReneeBornstein committed:
Update firepower-restapi-103.json

removed dashes in titles

ReneeBornstein committed:
Update firepower-restapi-104.json

changed above "title" capitalization and sentence structure
removed dashes in "files" titles
checked all spelling

ReneeBornstein committed:
Update firepower-restapi-105.json

removed dashes in titles

annegentle committed:
Merge branch 'master' into LL-firepower-rest-api105-JSON
